frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -67,5 +69,20 @@ public Car getCar(@PathVariable("carId") int carId) {
         }
         return map;
     }
+
+   @DeleteMapping(path = "car/{carId}")
+    public Map<Integer, Car> deleteCar(@PathVariable int carId) {
+            map.entrySet().removeIf(e -> carId == e.getKey());
```

Comment:
Not bad but a bit hard to understand. Can you explain `e -> carId`? I think there is an easier way to do this.

Reply:
Maps also have a delete key method